crates/bevy_scene/src/dynamic_scene.rs

```diff
@@ -28,8 +28,9 @@ pub struct DynamicScene {
 
 /// A reflection-powered serializable representation of an entity and its components.
 pub struct DynamicEntity {
-    /// The transiently unique identifier of a corresponding `Entity`.
-    pub entity: u32,
+    /// The identifier of the entity, unique within a scene (and the world it may have been generated from).
+    /// Components containing entitiy references will identify their referant via this identifier.
```

Comment:
> Components containing entitiy references will identify their referant via this identifier.

This is a little difficult to understand. What does it mean for a component to identify other entities identifiers using this entity identifier? I'm also not sure if you mean "referent" (the receiver of the entity) or "referrer" (the giver of the entity). I might suggest using simpler terms to describe what you mean here.